### DISCUSSION TITLE
Can't the steps for updating the program in nuxt3 be updated without feeling it?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#25657] [Questions]
### Describe the feature

Can't the steps for updating the program in nuxt3 be updated without feeling it?

Every time I update the program and overwrite the .output folder, before restarting PM2, when accessing the website, there will be an error with code 500.
How can I automatically update the CI CD correctly?

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by huang-julien** [comment]
Hey :wave: i'm not sure to understand the 500 response issue. if you have a server running and you overwrite the .output folder, the server won't be able to serve public files.

So you probably need slot swapping between your new and old instance
